### ISSUE TITLE
[cms-ci] CI Server at MAX load, tasks failing

### ISSUE THREAD

**Issue by ElijahLynn** [department-of-veterans-affairs/va.gov-cms#1832]
![image](https://user-images.githubusercontent.com/1504756/82695970-f57e4f80-9c1a-11ea-9a6f-be1dbec8fdf0.png)

**Comment by indytechcook** [comment]
I was watching the queries via innotop and all were normal queries (not insert/udpate) against the devshopt db:

```
When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00     8  6.47k     0  99/ 0/ 0/ 0       0.00%    100.00%  985.11k  17.85M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  328926  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '20216' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LIMIT 1
Query  329094  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '15004' OR t.rid = '15005') ORDER BY t.vid DESC LIMIT 20
Query  329123  statistics    devshopc  localhost      devshopcmsva  00:00  SELECT name, value FROM hosting_task_arguments WHERE vid = '53540'
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 6.02k QPS, 7/3/6 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00     7  6.02k     0  92/ 6/ 0/ 0       0.00%    100.00%  955.94k   9.92M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  328709  query end     devshopc  localhost      devshopcmsva  00:00  INSERT INTO hosting_task_log (vid, nid, type, message, error, timestamp) VALUES ('54920', '33099', 'debugnotify', 'Scanning into /var/aegir/projects/cms/pr1558/docroot/drush for /.*aliases\\.drus
Query  329094  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '21093' OR t.rid = '21094') ORDER BY t.vid DESC LIMIT 20
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 6.86k QPS, 10/8/3 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00    10  6.86k     0  97/ 2/ 0/ 0       0.00%    100.00%  1.04M  12.65M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State               User      Host           DB            Time   Query
Query  328709  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '15782' OR t.rid = '15783') ORDER BY t.vid DESC LIMIT 20
Query  329094  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '33040' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  329199  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17569' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'enable' ORDER BY t.vid DESC LI
Query  329226  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '16101' LIMIT 1
Query  329230  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '15288' OR t.rid = '15289') ORDER BY t.vid DESC LIMIT 20
Query  329263  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329264  Creating sort inde  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid FROM node n INNER JOIN hosting_task t ON n.vid=t.vid WHERE (type = 'task') AND( (task_status = '-1') OR (task_status = '0') )AND (n.nid NOT IN ('17253', '17253', '17253'
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 3.77k QPS, 7/4/5 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00     6  3.77k     0  99/ 0/ 0/ 0       0.00%    100.00%  579.19k   6.11M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  328709  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '24737' OR t.rid = '24738') ORDER BY t.vid DESC LIMIT 20
Query  329289  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '17546' LIMIT 1
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 6.82k QPS, 6/3/6 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00     7  6.82k     0  99/ 0/ 0/ 0       0.00%    100.00%  1.02M  12.81M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  329320  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '15880' OR t.rid = '15881') ORDER BY t.vid DESC LIMIT 20
Query  329356  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '13899' OR t.rid = '13900') ORDER BY t.vid DESC LIMIT 20
Query  329364  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 5.93k QPS, 7/3/5 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00     7  5.93k     0  99/ 0/ 0/ 0       0.00%    100.00%  904.98k   9.80M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  329320  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '30605' LIMIT 1
Query  329408  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '15531' LIMIT 1
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 6.97k QPS, 10/7/3 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00    11  6.97k     0  99/ 0/ 0/ 0       0.00%    100.00%  1.04M  12.85M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State              User      Host           DB            Time   Query
Query  329408  Sending to client  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329450  Sending data       devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17569' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LIM
Query  329455  Sending data       devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '15881' LIMIT 1
Query  329464  Sending to client  devshopc  localhost      devshopcmsva  00:00  SELECT alias FROM url_alias WHERE source = 'node/15977' AND language IN ('en', 'und') ORDER BY language ASC, pid DESC
Query  329496  Sending data       devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '19470' LIMIT 1
Query  329503  starting           devshopc  localhost      devshopcmsva  00:00  SELECT https_enabled, client_authentication, client_authentication_path FROM hosting_https_site WHERE vid = '54917'
Query  329504  starting           devshopc  localhost      devshopcmsva  00:00  SELECT https_enabled, client_authentication, client_authentication_path FROM hosting_https_site WHERE vid = '54708'
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 6.89k QPS, 12/5/2 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00    12  6.89k     0  99/ 0/ 0/ 0       0.00%    100.00%  1.02M  11.10M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd      ID      State               User      Host           DB            Time   Query
Query    329408  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '32160' OR t.rid = '32161') ORDER BY t.vid DESC LIMIT 20
Query    329455  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '21121' OR t.rid = '21122') ORDER BY t.vid DESC LIMIT 20
Query    329511  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17561' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC
Query    329570  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT alias FROM url_alias WHERE source = 'node/17267' AND language IN ('en', 'und') ORDER BY language ASC, pid DESC
Connect  329574  Receiving from cli  unauthen  localhost                    00:00
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 6.55k QPS, 12/9/3 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn     BpsOut
Now    0.00    13  6.55k     0  99/ 0/ 0/ 0       0.00%    100.00%  1011.94k  12.17M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%     1.65M   4.62M

Cmd    ID      State               User      Host           DB            Time   Query
Query  329408  query end           devshopc  localhost      devshopcmsva  00:00  UPDATE hosting_devshop_github_pull_requests SET site_nid='20399', number='1577', pull_request_object='O:8:\"stdClass\":34:{s:3:\"url\";s:81:\"https://api.github.com/repos/department-of-vete
Query  329455  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '3116' OR t.rid = '3117') ORDER BY t.vid DESC LIMIT 20
Query  329571  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17557' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  329573  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17565' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'enable' ORDER BY t.vid DESC LI
Query  329580  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '17561' OR t.rid = '17562') ORDER BY t.vid DESC LIMIT 20
Query  329616  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT iid FROM hosting_package_instance i left join hosting_package p on p.nid=i.package_id WHERE p.package_type = 'platform' AND i.rid = '17558'
Query  329636  Creating sort inde  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid FROM node n INNER JOIN hosting_task t ON n.vid=t.vid WHERE (type = 'task') AND( (task_status = '-1') OR (task_status = '0') )AND (n.nid NOT IN ('17253', '17253', '17253'
Query  329637  Creating sort inde  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid FROM node n INNER JOIN hosting_task t ON n.vid=t.vid WHERE (type = 'task') AND( (task_status = '-1') OR (task_status = '0') )AND (n.nid NOT IN ('17253', '17253', '17253'
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 4.33k QPS, 7/4/5 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00     7  4.33k     0  98/ 0/ 0/ 0       0.00%    100.00%  680.14k   7.87M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State         User      Host           DB            Time   Query
Query  329408  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '17427' OR t.rid = '17428') ORDER BY t.vid DESC LIMIT 20
Query  329698  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329699  Sending data  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 5.83k QPS, 12/9/0 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn    BpsOut
Now    0.00    12  5.83k     0  99/ 0/ 0/ 0       0.00%    100.00%  893.14k  10.85M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%    1.65M   4.62M

Cmd    ID      State               User      Host           DB            Time   Query
Query  329709  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17569' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  329711  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329723  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17565' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  329754  starting            devshopc  localhost      devshopcmsva  00:00  SELECT task_type, executed, delta, rid, task_status, r.type as ref_type FROM hosting_task t LEFT JOIN node r ON t.rid = r.nid WHERE t.vid = '37132'
Query  329767  statistics          devshopc  localhost      devshopcmsva  00:00  SELECT alias FROM url_alias WHERE source = 'node/31934' AND language IN ('en', 'und') ORDER BY language ASC, pid DESC
Query  329768  starting            devshopc  localhost      devshopcmsva  00:00  SELECT revision.vid AS vid, base.uid AS uid, revision.title AS title, revision.log AS log, revision.status AS status, revision.comment AS comment, revision.promote AS promote, revision.stic
Query  329769  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329770  Creating sort inde  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid FROM node n INNER JOIN hosting_task t ON n.vid=t.vid WHERE (type = 'task') AND( (task_status = '-1') OR (task_status = '0') )AND (n.nid NOT IN ('17253', '17253', '17253'
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 6.55k QPS, 7/2/5 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00     7  6.55k     0  97/ 1/ 0/ 0       0.00%    100.00%  3.28M  10.76M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State              User      Host           DB            Time   Query
Query  329711  Sending data       devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '26469' OR t.rid = '26470') ORDER BY t.vid DESC LIMIT 20
Query  329842  Sending to client  devshopc  localhost      devshopcmsva  00:00  SELECT alias FROM url_alias WHERE source = 'node/20446' AND language IN ('en', 'und') ORDER BY language ASC, pid DESC
[RO] Query List (? for help)                                                                                                                                                                                        localhost, 3d, 6.30k QPS, 17/13/0 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00    17  6.30k     0  98/ 0/ 0/ 0       0.00%    100.00%  1.62M  13.07M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State               User      Host           DB            Time   Query
Query  329711  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '20216' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'enable' ORDER BY t.vid DESC LI
Query  329855  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17557' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  329862  Sending data        pr1827ci  localhost      pr1827cicmsv  00:00  SELECT cid, data, created, expire, serialized, tags, checksum FROM cache_config WHERE cid IN ( 'core.base_field_override.menu_link_content.va-erie-health-care.id', 'core.base_field_override
Query  329864  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '16087' LIMIT 1
Query  329875  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '16087' LIMIT 1
Query  329880  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '14976' OR t.rid = '14977') ORDER BY t.vid DESC LIMIT 20
Query  329892  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT ta.nid FROM hosting_task_arguments ta INNER JOIN hosting_task t ON ta.nid = t.nid WHERE name = 'target_platform' AND value = '12791' LIMIT 1
Query  329913  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid, n.type AS type FROM node n WHERE (type IN ('site', 'platform', 'project', 'server'))
Query  329914  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT hl.* FROM hosting_logs hl WHERE (hl.site_nid IN ('33113'))
Query  329915  Opening tables      devshopc  localhost      devshopcmsva  00:00  SELECT nid, name AS hosting_name FROM hosting_context WHERE nid IN('31862')
Query  329916  Sending to client   devshopc  localhost      devshopcmsva  00:00  SELECT nid, name AS hosting_name FROM hosting_context WHERE nid IN('19491')
Query  329917  Creating sort inde  devshopc  localhost      devshopcmsva  00:00  SELECT n.nid AS nid FROM node n INNER JOIN hosting_task t ON n.vid=t.vid WHERE (type = 'task') AND( (task_status = '-1') OR (task_status = '0') )AND (n.nid NOT IN ('17253', '17253', '17253'
[RO] Query List (? for help)                                                                                                                                                                                          localhost, 3d, 6.18k QPS, 5/2/8 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00     5  6.18k     0  97/ 1/ 0/ 0       0.00%    100.00%  4.38M  12.01M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State       User      Host           DB            Time   Query
Query  329862  statistics  pr1827ci  localhost      pr1827cicmsv  00:00  SELECT menu_tree.* FROM menu_tree menu_tree WHERE id IN ('entity.node_type.collection')
[RO] Query List (? for help)                                                                                                                                                                                         localhost, 3d, 6.90k QPS, 11/5/4 con/run/cac thds, 5.7.28

When   Load  Cxns  QPS    Slow  Se/In/Up/De%  QCacheHit  KCacheHit  BpsIn  BpsOut
Now    0.00     9  6.90k     0  97/ 1/ 0/ 0       0.00%    100.00%  3.39M  13.85M
Total  0.00   151  1.69k    34  94/ 3/ 0/ 0       0.00%     71.45%  1.65M   4.62M

Cmd    ID      State               User      Host           DB            Time   Query
Query  329880  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid, t.task_type FROM node tn INNER JOIN hosting_task t ON tn.nid = t.nid WHERE (t.rid = '16293' OR t.rid = '16294') ORDER BY t.vid DESC LIMIT 20
Query  329993  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17565' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'enable' ORDER BY t.vid DESC LI
Query  329998  Sending data        devshopc  localhost      devshopcmsva  00:00  SELECT t.nid FROM hosting_task t INNER JOIN node n ON t.vid = n.vid WHERE t.rid = '17569' AND (t.task_status = '0' OR t.task_status = '-1') AND t.task_type = 'delete' ORDER BY t.vid DESC LI
Query  330036  Creating tmp table  devshopc  localhost      devshopcmsva  00:00  SELECT DISTINCT(language) FROM hosting_package_languages WHERE iid = '544879'
```

**Comment by indytechcook** [comment]
I want to follow up and take some of these queries to do explains/plans on them to see if they are using indexes correctly.

**Comment by ElijahLynn** [comment]
We think this was due to the general performance of the dashboard pages loading as there was only one task running at the time. We stopped the server, boosted from 8 vCPU to 36 vCPU for now until the real issue is resolved, which is suspected to be #1232. 

And we have a follow up issue to move from on-instance MySQL/MariaDB to #1689.
